control-board/src/tasks/control_task.rs

```diff
@@ -298,16 +331,22 @@ impl <
                     self.last_accel_x_ms = accel_ms[0];
                     self.last_accel_y_ms = accel_ms[1];
                 }
+
+                while let Some(kicker_telemetry) = self.kicker_telemetry_subscriber.try_next_message_pure() {
+                    self.last_kicker_telemetry = kicker_telemetry;
+                }
+
+                while let Some(power_telemetry) = self.power_telemetry_subscriber.try_next_message_pure() {
```

Comment:
Should we check if power is inop and stop if so?

Reply:
No, the power telemetry is non-critical. We shouldn't stop mid match if the connection blips out imo, but I'm open to being convinced otherwise. This would also prevent bench top testing on power supply not going thru the power board. I tried (and I think succeeded) to make sure that if anything is powered on bench supply, it may set warnings/errors but no subsystem will stop. There probably some wisdom in stopping wheels/charging but I thought the risk was worth it for fast iteration and/or midmatch debugging. Def worth a convo tho. We can kinda tell when we're on bench top supply because no balance connector, but that's also indistinguisable from forgetting to plug it in.